docs/02-guides/extend-the-graphql-schema.mdx

```diff
@@ -8,9 +8,284 @@ sidebar_position: 3
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
 
-See
-https://developers.front-commerce.com/docs/2.x/essentials/extend-the-graphql-schema
+When developing an e-commerce store, you might at some point need to expose
+new data in your unified GraphQL schema to support new features and allow
+frontend developers to use them. Front-Commerce’s GraphQL modules is the
+mechanism allowing to extend and override any part of the schema defined by
+other modules.
+
+The Front-Commerce core and platform integrations (such as Magento2) are
+implemented as GraphQL modules. They leverage features from the GraphQL Schema
+Definition Language (<abbr title="Schema Definition Language">SDL</abbr>).
+
+This page will guide you through the process of exposing a new feature in your
+GraphQL schema. We will create an extension with a  GraphQL module that allows
+to maintain a counter of clicks for a product.
+
+## Create an extension
```

Comment:
**thought:** Should this section be just a link to the `Register an extension` guide, or should both of them be merged at some point?